src/assets/js/setup-dropdown-menu.js

```diff
@@ -0,0 +1,41 @@
+/* global document */
+(function setupDropdownMenu() {
+  const menu = document.querySelector('.DropdownMenu');
+
+  if (menu) {
+    menu.addEventListener('click', (event) => {
```

Comment:
Could all of this be done on the delegated eventListener?

Reply:
ah yes, thanks. This simplifies things a bit.